src/NUnitConsole/nunit-console/Program.cs

```diff
@@ -21,16 +21,14 @@
 // WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 // ***********************************************************************
 
-using System;
-using System.IO;
-using System.Reflection;
-using Mono.Options;
-using NUnit.Common;
-using NUnit.Engine;
-
 namespace NUnit.ConsoleRunner
 {
-    using Utilities;
+    using System;
+    using System.IO;
+    using System.Reflection;
+    using Common;
+    using Engine;
+    using Mono.Options;
```

Comment:
We keep using statements outside the namespace in each file except for immediate descendants of the enclosing namespace.